server/reports/attendance/attendance-endpoint.js

```diff
@@ -109,8 +109,7 @@ export function attendanceRequestHandler(request, response) {
             let fields = ["class", "capacity", "registered", "attended"];
             let parser = new J2C.Parser({ fields });
             let csv = parser.parse(attendanceReport);
-            let fileName = "AttendanceReport.csv";
-            //todo: we should choose a filename that includes helpful info like the date range
+            let fileName = "Attendance " + startdate + "-" + enddate + ".csv";
```

Comment:
I was doubtful that the dates would work as filenames because they contain / characters which are often illegal. But... on Mac firefox and Chrome, they got autoreplaced with _ characters somehow. Worked at least there.

Reply:
This wont work on windows, as windows allows filenames with spaces.